hippunfold/workflow/rules/resample_final_to_crop_t1.smk

```diff
@@ -70,6 +70,19 @@ rule resample_coords_native_crop:
         'ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS={threads} '
         'antsApplyTransforms -d 3 --interpolation NearestNeighbor -i {input.nii} -o {output.nii} -r {input.ref}  -t [{input.xfm},1]' 
 
+rule resample_isovol_native_crop:
```

Comment:
this rule could be left out if we instead update `resample_coords_native_crop` to add `desc` wildcard

Reply:
fixed